app.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const {getAllTopics} = require ('./controllers/topics-controller')
+
+// app.use(express.json())
```

Comment:
Please delete this until it is needed. I know it is commented out, but we should only be adding enough code to complete the brief on each ticket